### ISSUE TITLE
strong tag

### ISSUE THREAD

**Issue by rimmesbe** [leo-cavalcanti/prj-rev-bwfs-dasmoto#1]
Might be a good place for a strong tag instead of a span. Get that bold setting default.

https://github.com/leo-cavalcanti/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L14

**Comment by leo-cavalcanti** [comment]
Thank you for this and all other comments. I understood why is good to use strong instead of span, but there is another property that should be applied to that part, that is the font color blue. 
Or are you telling me to continue using the span to set the color at the same time that I use strong?

**Comment by leo-cavalcanti** [comment]
L.14 would be like this:
      <p>Made of the highest quality oak, Hacksaw brushes are known for their weight and ability to hold paint in large amounts. Available in different sizes.<span class="price"> <strong>Starting at $3.00 / brush.</strong></span></p>

on the css:

.price {
  color: blue;
}

**Comment by rimmesbe** [comment]
Yes you would still use the price class, but you wouldn't need the bold
styling if you use strong tag. Also, .price is ok for a class name in this
project because it is describing the section, but alternatively you can
create class names that describe the styling they apply. This makes them
even more reusable:
.color-blue, .font-blue, etc
